com.unity.multiplayer.mlapi/Tests/Runtime/Components/NetworkVariableTestComponent.cs

```diff
@@ -109,8 +109,8 @@ private void InitializeTest()
 
             // NetworkVariable NetworkVariableSettings Constructor Test Coverage
             var settings = new NetworkVariableSettings();
-            settings.ReadPermission = NetworkVariablePermission.ServerOnly;
-            settings.WritePermission = NetworkVariablePermission.ServerOnly;
+            settings.ReadPermission = NetworkVariableReadPermission.OwnerOnly;
+            settings.WritePermission = NetworkVariableWritePermission.ServerOnly;
```

Comment:
Does that put the netvar in a situation where the server can only write, but maybe not read if it is not the owner? Or do write permission also come with implicit read permission?

Reply:
Wouldn't we want everyone capable of reading and only place restrictions on the writing aspect of NetworkVariables?